src/ngx_http_lua_socket_tcp.c

```diff
@@ -843,11 +843,13 @@ ngx_http_lua_socket_tcp_bind(lua_State *L)
     u_char               *text;
     size_t                len;
     ngx_addr_t           *local;
+    int                   port;
 
     n = lua_gettop(L);
 
-    if (n != 2) {
-        return luaL_error(L, "expecting 2 arguments, but got %d",
+    // 修正参数检查，允许2或3个参数
```

Comment:
Please use English

Reply:
Please use the C style comment /* */

Reply:
use the ngx-releng tool from [openresty-devel repo](https://github.com/openresty/openresty-devel-utils)